core/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM rust:1.70.0 as core
+
+WORKDIR /app
+
+COPY . .
+
+RUN cargo build --release
+
+EXPOSE 3001
```

Comment:
Is this needed ?

Reply:
It's in most dockerfiles I think we need to say which port we expose no?